datalad/core/distributed/tests/test_clone.py

```diff
@@ -573,6 +573,13 @@ def test_expanduser(srcpath, destpath):
     src = Dataset(Path(srcpath) / 'src').create()
     dest = Dataset(Path(destpath) / 'dest').create()
 
+    # We switch away from home set up in datalad.setup_package(), so make sure
+    # we have a valid identity.
+    with open(op.join(srcpath, ".gitconfig"), "w") as fh:
+        fh.write("[user]\n"
+                 "name = DataLad oooooTester\n"
```

Comment:
Hmm, those o's were from local debugging, and I didn't really mean to keep them around.  Oh well :)